src/main/java/frc/robot/Robot.java

```diff
@@ -86,6 +89,11 @@ public void autonomousPeriodic() {
    */
   @Override
   public void teleopPeriodic() {
+    double xAxisDemand = drive.driveController.getX(Hand.kRight) * -.3; //get joystick values
+    double yAxisDemand = drive.driveController.getY(Hand.kLeft) * -1;
+    boolean turnStickButton = drive.driveController.getStickButton(Hand.kRight); //this is the button taht is pressed if you push down on the joysticks, or in this case the right joystick
+    boolean toggleY = drive.driveController.getYButton(); //gets raw y button value, toggle stuff is all done in the arcade drive function
```

Comment:
Relevant to comment in Drive class. Do we want one of the controllers to exist in the drive class? Or should it only pass in what is necessary (joystick values and fucntion calls to shift) to Drive class to accomplish its function?

Reply:
discussed in person